README.md

```diff
@@ -527,7 +527,28 @@ The database name for the create script gets resolved in the following manner:
 1. Package name.
 > Note: 
 >- the generated script also uses the resolved database name via a setvar command.
->- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`   
+>- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`
+
+## Static code analysis
+Starting with version 2.6.2 of the SDK, there is support for running static code analysis during build. The SDK includes the following sets of rules:
+
+- Microsoft.Rules ([1](https://learn.microsoft.com/previous-versions/visualstudio/visual-studio-2010/dd193411), [2](https://learn.microsoft.com/previous-versions/visualstudio/visual-studio-2010/dd193246) and [3](https://learn.microsoft.com/previous-versions/visualstudio/visual-studio-2010/dd172117))
+- [SqlServer.Rules](https://github.com/tcartwright/SqlServer.Rules/blob/master/docs/table_of_contents.md)
+- [Smells](https://github.com/davebally/TSQL-Smells)
+
+Static code analysis can be enabled by adding the following to the project file:
+
+```xml
+<Project Sdk="MSBuild.Sdk.SqlProj/2.6.2">
+  <PropertyGroup>
+    <RunSqlCodeAnalysis>True</RunSqlCodeAnalysis>
+    <CodeAnalysisRules>-SqlServer.Rules.SRD0006;-Smells.*</CodeAnalysisRules>
+  </PropertyGroup>
+</Project>
+```
+The optional `CodeAnalysisRules` property allows you to disable individual rules or groups of rules.
+
+A xml file with the analysis results is created in the output folder, and any problems found during analysis are reported as build warnings.
```

Comment:
Does this work with `TreatWarningsAsErrors`? I think we might want to write something about that here?

Reply:
Let me test

Reply:
Perhaps we should call that out in the doc and then add that later if there's a demand for it?

Reply:
Added a note

Reply:
@jmezach Do you think I should add more?

Reply:
I think this is fine for now.